src/components/header/index.tsx

```diff
@@ -1,39 +1,100 @@
 import React, { memo, FC, useState } from 'react';
 import { compose } from 'redux';
+import { Link as RouteLink } from 'react-router-dom';
 
 import { Trigger, Menu } from '@fellesdatakatalog/dropdown-menu';
+
+import Translation from '../translation';
+
 import env from '../../env';
 
 import SC from './styled';
 
-const { FDK_REGISTRATION_BASE_URI, ADMIN_GUI_BASE_URI } = env;
+import {
+  withTranslations,
+  Props as TranslationsProps,
+  Language
+} from '../../providers/translations';
+
+const { USE_DEMO_LOGO } = env;
 
-const Header: FC = () => {
+interface Props extends TranslationsProps {}
+
+const Header: FC<Props> = ({ translationsService }) => {
   const [isDropdownMenuOpen, setIsDropdownMenuOpen] = useState(false);
+  const [isLanguageMenuOpen, setIsLanguageMenuOpen] = useState(false);
 
   const openDropdownMenu = () => setIsDropdownMenuOpen(true);
   const closeDropdownMenu = () => setIsDropdownMenuOpen(false);
 
+  const openLanguageMenu = () => setIsLanguageMenuOpen(true);
+  const closeLanguageMenu = () => setIsLanguageMenuOpen(false);
+
+  const handleChangeLanguage = (language: Language) => {
+    translationsService.changeLanguage(language);
+    closeLanguageMenu();
+  };
+
   return (
     <SC.Header>
       <SC.Row>
-        <SC.Logo />
+        <RouteLink to='/'>
+          {USE_DEMO_LOGO ? <SC.DemoLogo /> : <SC.Logo />}
+        </RouteLink>
         <SC.NavigationLinks>
           <li>
-            <SC.Link href={FDK_REGISTRATION_BASE_URI}>Registrere data</SC.Link>
+            <SC.Link href='/about'>
+              <Translation id='header.about' />
+            </SC.Link>
           </li>
           <li>
-            <SC.Link href={ADMIN_GUI_BASE_URI}>Høste data</SC.Link>
+            <SC.Link href='/organizations'>
+              <Translation id='header.organizations' />
+            </SC.Link>
           </li>
           <li>
-            <SC.Link href='/publishing/terms-of-use'>Bruksvilkår</SC.Link>
+            <SC.Link href='/reports'>
+              <Translation id='header.reports' />
+            </SC.Link>
           </li>
           <li>
-            <SC.Link href='/' target='_self' external>
-              Søk i Felles datakatalog
+            <SC.Link href='/publishing' target='_self' external>
+              <Translation id='header.publishing' />
             </SC.Link>
           </li>
+          <li>
+            <SC.LanguageMenu
+              isOpen={isLanguageMenuOpen}
+              onClose={closeLanguageMenu}
+            >
+              <Trigger>
+                <SC.MenuButton onClick={openLanguageMenu}>
+                  <Translation id='lang.chosenLanguage' />
+                </SC.MenuButton>
+              </Trigger>
+              <Menu>
+                <SC.Menu>
+                  <li>
+                    <SC.Link onClick={() => handleChangeLanguage(Language.NB)}>
```

Comment:
Se denne kommentar sammen med forslag til `handleChangeLanguage`

```suggestion
                    <SC.Link onClick={handleChangeLanguage(Language.NB)}>
```